attest/ake/src/responder.rs

```diff
@@ -82,13 +82,19 @@ impl ResponderTransitionMixin for Start {
     fn handle_response<KexAlgo, Cipher, DigestAlgo>(
         csprng: &mut (impl CryptoRng + RngCore),
         handshake_state: HandshakeState<KexAlgo, Cipher, DigestAlgo>,
-        ias_report: VerificationReport,
+        attestation_evidence: EvidenceKind,
     ) -> Result<(Ready<Cipher>, AuthResponseOutput), Error>
     where
         KexAlgo: Kex,
         Cipher: NoiseCipher,
         DigestAlgo: NoiseDigest,
     {
+        // TODO: This will be replaced with dcap evidence serialization.
+        //       This code will never run.
```

Comment:
Why would this never run?

Reply:
I understand why we don't currently need to handle the DCAP variant here (thanks to @nick-mobilecoin for pointing out this code only runs in the enclaves), but I am not sure what `this code will never run` implies.

Reply:
Sorry, not the best explanation on my part. This code is running in tests/workflows right now, but it will never run in production. By that point it will be replaced with dcap logic. This comment is more of a TODO for myself so I know where to look later